R/start_addin.R

```diff
@@ -0,0 +1,85 @@
+#' Addin for starting develop mode.
+#'
+#' Allows users to see list of orderly reports and choose one to open in
+#' development mode.
+#'
+#' @export
+start_addin <- function() {
+
+  # Our ui will be a simple gadget page, which
+  # simply displays the time in a 'UI' output.
+  ui <- miniUI::miniPage(
+    miniUI::gadgetTitleBar("Choose report to start development",
+                           left = NULL),
+    miniUI::miniContentPanel(
+      DT::dataTableOutput("reports")
+    )
+  )
+
+  server <- function(input, output, session) {
+
+    report_list <- list_reports()
+    output$reports <- DT::renderDataTable(
+      DT::datatable(report_list[, c("report", "modified")],
+      rownames = FALSE,
+      height = "100%",
+      selection = "none",
+      callback = htmlwidgets::JS(
+        "table.on('click.dt', 'td', function() {
+            var row_=table.cell(this).index().row;
+           Shiny.onInputChange('row', row_ + 1 );
+        });"),
+      options = list(
+        paging = FALSE,
+        scrollResize = TRUE,
+        scrollY = 250,
+        scrollCollapse = TRUE)
+      ) %>%
```

Comment:
You don't _need_ this, though it's fine if you want to keep it

Reply:
gone, thanks